src/panels/lovelace/hui-root.ts

```diff
@@ -554,10 +587,12 @@ class HUIRoot extends LitElement {
 
   private _enableEditMode(): void {
     this.lovelace!.setEditMode(true);
+    fireEvent(this, "iron-resize");
```

Comment:
Does this work? Once we set editMode it will take a few event loop cycles before this element is updated. 

I think that we should put firing this event in `updated` and fire it whenever edit mode changes.

Reply:
This works well as is, without it going into edit mode causes the header to overlap and coming out causes the space taken up by the 2 rows to remain. Will make the change though.